tests/e2e/specs/shopper/mini-cart.test.js

```diff
@@ -26,6 +28,19 @@ const clickMiniCartButton = async () => {
 	await page.click( '.wc-block-mini-cart__button' );
 };
 
+const WooCommerce = new WooCommerceRestApi( {
+	url: `${ process.env.WORDPRESS_BASE_URL }/`,
+	consumerKey: 'consumer_key', // Your consumer key
+	consumerSecret: 'consumer_secret', // Your consumer secret
```

Comment:
Shouldn't these be `env` variables?

Reply:
I added a comment explaining why we use `consumerKey` and `consumerSecret` that way.

Reply:
Nice, thank you for clarifying! Perhaps removing those comments (`// Your consumer key`) which seem they might have been copy+pasted from docs and are perhaps a bit confusing would be the last suggestion I've got :)